spec/std/process/find_executable_spec.cr

```diff
@@ -0,0 +1,138 @@
+require "spec"
+require "../../support/tempfile"
+
+{% if flag?(:win32) %}
+  def find_executable_test_files
```

Comment:
I'd suggest to use constants instead of methods for this, because that's what this is: constant values.

Reply:
Oh yeah, I was trying something else where it made sense